### ISSUE TITLE
Support for async dynamic attributes

### ISSUE THREAD

**Issue by radekmie** [meteor/blaze#443]
As I wrote in #428, async dynamic attributes (i.e., `<img {{getAttributesAsync}}>`) is not supported yet. I believe it should be enough to cover it with something like [`waitForAllAttributesAndContinue`](https://github.com/meteor/blaze/blob/c5e2cac242f4889972fb253401fbea5ddbfaee4d/packages/blaze/materializer.js#L98) for standard attributes, but also lifted for arrays (i.e., support arrays of objects with `Promise`s).

**Comment by radekmie** [comment]
No, there are no "limitations". It's just a different part of code (not shared with singular attributes) and it'd require separate implementation. I have it partly working locally and plan to finish it by the end of the next week.

**Comment by jankapunkt** [comment]
@radekmie you can also create a draft PR and I can chime in if that makes sense
